articles/virtual-machine-scale-sets/spot-placement-score.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Spot Placement Score (Preview)
+description: Learn how to use Azure Spot Placement Score to evaluate deployment success.
+author: 
```

Comment:
Don't forget to add your Microsoft alias to "author" and your GH username to "ms.author".

Reply:
done, thanks